Entities/MOSRotating.cpp

```diff
@@ -542,23 +543,37 @@ int MOSRotating::Save(Writer &writer) const
 }
 
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Virtual method:  RemoveWounds
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Removes a specified amount of wounds and returns damage caoused by this wounds. 
-//					Head multiplier is not used.
+/// <summary>
+/// Attaches the passed in wound AEmitter and adds it to the list of wounds, changing its parent offset to the passed in Vector.
+/// </summary>
+/// <param name="pWound">The wound AEmitter to add</param>
+/// <param name="parentOffsetToSet">The vector to set as the wound AEmitter's parent offset</param>
+void MOSRotating::AddWound(AEmitter *pWound, const Vector & parentOffsetToSet)
```

Comment:
Is this fine without the giblimit check that was in AddEmitter? I'd think you'd need that stuff, especially with lua wound adding!

Reply:
Yup, definitely need that check.

Reply:
I'd rather you matched this to what it was before (I don't know if it'll make the lua binding a pain but I'm pretty sure you can do default values in luaman somehow it, we can figure it out if needed). I.e. 

```
	void MOSRotating::AddWound(AEmitter *pWound, const Vector & parentOffsetToSet, bool checkGibWoundLimit)
	{
		if (pWound)
		{
			if (checkGibWoundLimit && !ToDelete() && m_GibWoundLimit && m_Wounds.size() + 1 > m_GibWoundLimit)
			{
				// Indicate blast in opposite direction of emission
				// TODO: don't hardcode here; get some data from the emitter
				Vector blast(-5, 0);
				blast.RadRotate(pEmitter->GetEmitAngle());
				GibThis(blast);
			}
			else
			{
				pWound->Attach(this, parentOffsetToSet);
				m_Wounds.push_back(pWound);
			}
		}
	}
```

Reply:
Not sure if we need any of it.
The check arg was never used in the source, meaning all the blast stuff wasn't happening either.
Having the check arg set false in lua will allow you to add wounds past the GibWoundLimit and that can potentially jank stuff up.

Reply:
Yeah, if you look in the header the signature was `virtual void AttachEmitter(AEmitter *pEmitter, Vector emitOffset, bool checkGibWoundLimit = true);` which means checkGibWoundLimit defaulted to true when unspecified. If you mimic this in yours it should solve the problems.

And the blastwound stuff was happening anytime it was supposed to gib from adding wounds. It basically makes the gibbing directional based on the wound.